llvm/include/llvm/Analysis/DXILResource.h

```diff
@@ -117,6 +119,10 @@ class ResourceInfo {
 
   MSInfo MultiSample;
 
+  // We need a default constructor if we want to insert this in a MapVector.
+  ResourceInfo() {}
+  friend class MapVector<CallInst *, ResourceInfo>;
```

Comment:
Where is this being inserted in a MapVector? Is it DRM?

Reply:
It was, yes, but the latest version no longer needs this. Removed.